docs/addons/xray.md

```diff
@@ -1,5 +1,7 @@
 # AWS X-Ray Add-on
 
+This add-on is currently deprecated since the underlying manifests are incompatible with the latest versions of EKS. Please use [XRay Adot Add-on](xray-adot-addon.md).
```

Comment:
@elamaran11 this is the update that points to the adot addon.

Reply:
SGTM.